src/main/java/org/geppetto/core/manager/SharedLibraryManager.java

```diff
@@ -47,25 +47,26 @@ public synchronized static GeppettoLibrary getSharedCommonLibrary() throws Geppe
 			GeppettoPackage.Registry.INSTANCE.put(GeppettoPackage.eNS_URI, GeppettoPackage.eINSTANCE);
 
 			// We add all supported versions of the schema
-			String[] versions = new String[] { "v1.1.0" };
+
+			String[] versions = new String[] { "feature/30" };
```

Comment:
What about directly having v1.2.0 so to avoid the annoyance to change it again after the merge?

Reply:
It has to match with what's in the XMI of the common library. Which in turns has to be an existing tag or branch and 1.2.0 won't exist until we merge and release model.

Reply:
On the Python side is quite different: it is not really looking for the url, or versions, these are more placeholders that we can set as we like provided that we keep them consistent through the code

Reply:
Yeah, it's more constrained in Java and if you change that tests won't work.